app/src/main/java/de/westnordost/streetcomplete/quests/defibrillator/AddDefibrillatorLocation.kt

```diff
@@ -0,0 +1,38 @@
+package de.westnordost.streetcomplete.quests.defibrillator
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.LIFESAVER
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddDefibrillatorLocation : OsmFilterQuestType<DefibrillatorLocation>() {
+
+    override val elementFilter = """
+        nodes with
+        emergency = defibrillator
+        and !location and !defibrillator:location
+        and access !~ private|no"
+    """
+    override val changesetComment = "Specify defibrillator location"
+    override val wikiLink = "Tag:emergency=defibrillator"
+    override val icon = R.drawable.ic_quest_defibrillator
+    override val isDeleteElementEnabled = false
+    override val achievements = listOf(LIFESAVER)
+
+    override fun getTitle(tags: Map<String, String>) = R.string.quest_defibrillator_location
+
+    override fun getHighlightedElements(element: Element, getMapData: () -> MapDataWithGeometry) =
+        getMapData().filter("nodes with emergency = defibrillator")
+
+    override fun createForm() = AddDefibrillatorLocationForm()
+
+    override fun applyAnswerTo(answer: DefibrillatorLocation, tags: Tags, geometry: ElementGeometry, timestampEdited: Long) {
+        tags["defibrillator:location"] = answer.location
+    }
+}
+
+data class DefibrillatorLocation(val location: String)
```

Comment:
Any reason why this is not just a string?

Reply:
Not really. I see, this might work with simple String.